Formula/slackcat.rb

```diff
@@ -17,47 +17,42 @@ class Slackcat < Formula
 
   go_resource "github.com/bluele/slack" do
     url "https://github.com/bluele/slack.git",
-      :revision => "6d00f93158acefc3a0f605c171d1baa80ba86b73"
+      :revision => "ffdcd19858d03d5ebabba5bead2b5dfb18b2c73f"
   end
 
   go_resource "github.com/codegangsta/cli" do
     url "https://github.com/codegangsta/cli.git",
-      :revision => "c31a7975863e7810c92e2e288a9ab074f9a88f29"
+      :revision => "1efa31f08b9333f1bd4882d61f9d668a70cd902e"
   end
 
   go_resource "github.com/fatih/color" do
     url "https://github.com/fatih/color.git",
-      :revision => "9aae6aaa22315390f03959adca2c4d395b02fcef"
+      :revision => "87d4004f2ab62d0d255e0a38f1680aa534549fe3"
+  end
+
+  go_resource "github.com/mattn/go-colorable" do
+    url "https://github.com/mattn/go-colorable.git",
+      :revision => "9056b7a9f2d1f2d96498d6d146acd1f9d5ed3d59"
   end
 
   go_resource "github.com/mattn/go-isatty" do
     url "https://github.com/mattn/go-isatty.git",
       :revision => "56b76bdf51f7708750eac80fa38b952bb9f32639"
   end
 
-  go_resource "github.com/mattn/go-colorable" do
-    url "https://github.com/mattn/go-colorable.git",
-      :revision => "3dac7b4f76f6e17fb39b768b89e3783d16e237fe"
-  end
-
   go_resource "github.com/skratchdot/open-golang" do
     url "https://github.com/skratchdot/open-golang.git",
-      :revision => "c8748311a7528d0ba7330d302adbc5a677ef9c9e"
+      :revision => "75fb7ed4208cf72d323d7d02fd1a5964a7a9073c"
   end
 
   def install
-    ENV["GOBIN"] = bin
     ENV["GOPATH"] = buildpath
-    ENV["GOHOME"] = buildpath
-
-    mkdir_p buildpath/"src/github.com/vektorlab/"
-    ln_sf buildpath, buildpath/"src/github.com/vektorlab/slackcat"
     Language::Go.stage_deps resources, buildpath/"src"
-
-    system "go", "build", "-ldflags", "-s -X main.version=#{version}", "-o", bin/"slackcat"
+    system "go", "build", "-o", bin/"slackcat",
+           "-ldflags", "-X main.version=#{version}"
```

Comment:
Generally, no. We align things with the first "argument" after `system` where practical. If it makes you happier:
```ruby
 system "go", "build",
        "-o", bin/"slackcat",
        "-ldflags", "-X main.version=#{version}"
```
Could also be valid.